docs/Using Fleet/MDM-OS-updates.md

```diff
@@ -26,6 +26,10 @@ When a minimum version is enforced, the end users see a native macOS notificatio
 
 If the host was turned off when the deadline passed, the update will be scheduled an hour after it’s turned on.
 
+For Apple hosts (iOS 17.0+, iPadOS 17.0+, and macOS 14.0+) that use Automated Device Enrollment
```

Comment:
@gillespi314 I moved this under the macOS section, but now I'm not sure if this is implemented for all Apple hosts?  If so I think we'll probably want another section (iOS/iPadOS) under End user experience.